free-2.md

```diff
@@ -1,44 +1,57 @@
-# Free project 2
+# Free project 2 -- Wine pairings
 
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+This project would help wine enthusiasts choose wines that pair well with the food that they are eating or making. A language would be nice because wine pairings seem like something of a closed set of terms, which could be implemented in various ways.
```

Comment:
Interesting. As someone who knows nothing about this, I can see this being helpful in the right circumstances.